code/MathSpamProtectorField.php

```diff
@@ -21,17 +27,17 @@ class MathSpamProtectorField extends TextField
      * @var string
      */
     private static $question_prefix;
-    
+
     /**
      * @config
      *
      * @var bool $allow_numeric_answer
      */
     private static $allow_numeric_answer = true;
-    
+
     public function Field($properties = array())
     {
-        if (Config::inst()->get('MathSpamProtectorField', 'enabled')) {
+        if (Config::inst()->get('SilverStripe\SpamProtection\Maths\MathSpamProtectorField', 'enabled')) {
```

Comment:
We'd prefer to use `$this->config()->get('enabled')` in this instance

Reply:
I just copied the old code :)

Reply:
Ok, we can run over and update these later